README.md

```diff
@@ -220,13 +221,77 @@ Cloudant({account:me, password:password}, function(err, cloudant) {
 });
 ~~~
 
-After initialization, in general, callback functions receive three arguments:
+### Callback Signature
+
```

Comment:
I know it is in the examples later anyway, but I think it would be nice to have the function signature here too
```javascript
function(err, body, headers) {}
```

just so people can actually see the signature before the description of the parameters